webkitpy/benchmark_runner/browser_driver/linux_cogfdo_driver.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2018 Igalia S.L. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are
+# met:
+#
+#     * Redistributions of source code must retain the above copyright
+# notice, this list of conditions and the following disclaimer.
+#
+#     * Redistributions in binary form must reproduce the above
+# copyright notice, this list of conditions and the following disclaimer
+# in the documentation and/or other materials provided with the
+# distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+# OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+from linux_browser_driver import LinuxBrowserDriver
+
+class CogFdoBrowserDriver(LinuxBrowserDriver):
+    browser_name = 'cog-fdo'
+    process_search_list = ['cog']
+
+    def launch_url(self, url, options, browser_build_path):
+        self._browser_arguments.append('-P fdo')
+        self._browser_arguments = [url]
```

Comment:
This is replacing `self._browser_arguments`, isn't it? I believe that you need to `.append(url)` here, or use `.extend()` so you can put all in one line:

```python
    self._browser_arguments.extend(('--platform=fdo', url))
```

I also like it better passing the long version of the option (`--platform=fdo`), to make the code more self-documenting.

Reply:
yes, I catched it right now running the requested changes by Carlos. Good catch anyway,